std/uni.d

```diff
@@ -8888,8 +8900,11 @@ private dchar toTitlecase(dchar c)
     return c;
 }
 
-private alias UpperTriple = AliasSeq!(toUpperIndex, MAX_SIMPLE_UPPER, toUpperTab);
-private alias LowerTriple = AliasSeq!(toLowerIndex, MAX_SIMPLE_LOWER, toLowerTab);
+enum MAX_SIMPLE_UPPER = 1051;
+alias UpperTriple = AliasSeq!(toUpperIndex, MAX_SIMPLE_UPPER, toUpperTab);
+
+enum MAX_SIMPLE_LOWER = 1043;
+alias LowerTriple = AliasSeq!(toLowerIndex, MAX_SIMPLE_LOWER, toLowerTab);
```

Comment:
This is the only bit I couldn't figure out how to do lazily. Better ideas?

Reply:
Where are these aliases used?  Since they were originally private, you may be able to just rewrite the places where they're used to lazily instantiate the definition. Or maybe something like:
````
private alias UpperTriple() = ...
````
might do the trick? Not 100% this syntax is supported, or whether it will do what I think it will do.